pubpub/pubpub
# Connections UI updates

This PR contains the following changes (closes #1039):

- Removed placeholder thumbnail image (when external resource doesn’t have its own thumbnail).
- Add two new Pub-level options for displaying Pub Connections: default listing view (carousel/list), and an option to show the target resource's description by default.
- Better grouping of form controls in the Pub Connection editor.

_Test Plan_

- Create a Pub Connection and confirm both the dashboard views and editor view update correctly when toggling the new connection options.
- Ensure the UI tweaks match the designs in Figma.
